WorkflowFunctionBundle/Document/Authorization.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace OpenOrchestra\WorkflowFunctionBundle\Document;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+use OpenOrchestra\WorkflowFunction\Model\AuthorizationInterface;
+
+/**
+ * Description of Authorization
+ *
+ * @ODM\EmbeddedDocument
+ */
+class Authorization implements AuthorizationInterface
+{
+    /**
+     * @var string $name
+     *
+     * @ODM\Field(type="string")
+     */
+    protected $name;
```

Comment:
What do you think about referenceOne ContentType instead of name, @nicolasThal ?